ci/helm/preview/Chart.yaml

```diff
@@ -1,5 +1,16 @@
 apiVersion: v1
```

Comment:
Shouldn't we need to update the apiVersion to have the dependencies in this file?

Reply:
Nope, its' resolved client side when using Helm 3.